### PR TITLE
Feature/playback get imu sample

### PR DESCRIPTION
**Pull request by annStein**
For playback.py I added the method to get the next imu sample.

### CODE REVIEW

**Comment by lpasselin** [comment]
Sorry, I did not notice this PR until today! Thanks for the contribution, this looks well done.

@shagren it seems perfect but could you test?

File: pyk4a/playback.py [hunk 1]
```diff
@@ -178,6 +178,12 @@ def get_previouse_capture(self):
             thread_safe=self.thread_safe,
         )
 
+    def get_next_imu_sample(self) -> Optional["ImuSample"]:
```
Comment (shagren) [inline pyk4a/playback.py:181]: Please add
```python
from .pyk4a import ImuSample
```
before
```python
from .results import Result, StreamResult
```

Comment (shagren) [inline pyk4a/playback.py:181]: Please add
```python
from .pyk4a import ImuSample
```
before
```python
from .results import Result, StreamResult
```

File: pyk4a/pyk4a.cpp [hunk 2]
```diff
@@ -1312,6 +1312,31 @@ static PyObject *playback_get_previous_capture(PyObject *self, PyObject *args) {
...
+  _gil_restore(thread_state);
+
+  if (result != K4A_STREAM_RESULT_SUCCEEDED) {
+    return Py_BuildValue("IN", result, Py_None);
+  }
+  return Py_BuildValue("I{s:f,s:(fff),s:L,s:(fff),s:L}", result, "temperature", imu_sample.temperature, "acc_sample",
+                         imu_sample.acc_sample.xyz.x, imu_sample.acc_sample.xyz.y, imu_sample.acc_sample.xyz.z,
+                         "acc_timestamp", imu_sample.acc_timestamp_usec, "gyro_sample", imu_sample.gyro_sample.xyz.x,
+                         imu_sample.gyro_sample.xyz.y, imu_sample.gyro_sample.xyz.z, "gyro_timestamp",
+                         imu_sample.gyro_timestamp_usec);
```
Comment (shagren) [inline pyk4a/pyk4a.cpp:1337]:
```suggestion
                       imu_sample.acc_sample.xyz.x, imu_sample.acc_sample.xyz.y, imu_sample.acc_sample.xyz.z,
                       "acc_timestamp", imu_sample.acc_timestamp_usec, "gyro_sample", imu_sample.gyro_sample.xyz.x,
                       imu_sample.gyro_sample.xyz.y, imu_sample.gyro_sample.xyz.z, "gyro_timestamp",
                       imu_sample.gyro_timestamp_usec);
```

Just remove 2 spaces on these lines

**Review by shagren — CHANGES_REQUESTED** [review]
Great, just two small changes for better quality

**Comment by annStein** [comment]
Sorry, now I didn't see your comments for a long time. Hopefully this can be closed soon now. :)

**Review by shagren — APPROVED** [review]
LGTM

**Pull request merged**